src/cmd/cmd.zig

```diff
@@ -572,14 +570,14 @@ fn initRepair(
     };
 }
 
+fn runGossip(gossip_service: *GossipService) !void {
```

Comment:
few things:
- can we rename this to `runGossipWithConfigValues` or something more descriptive? 
- with this change we can remove all `spawnGossip` calls to just spawn the thread directly (vs single-line fn call)
- lastly, can we change `current` variable name to `gossip_config`

Reply:
can do